src/keywords.md

```diff
@@ -0,0 +1,12 @@
+## Keywords
+
+V is a very small language so it has less keywords. There are around 25 keywords.

```

Comment:
maybe change `less` to `fewer` maybe better

Reply:
`few` would sound better imo.

Reply:
ya... SGTM  👌